vllm/engine/arg_utils.py

```diff
@@ -1224,7 +1224,9 @@ def _is_v1_supported_oracle(self, model_config: ModelConfig) -> bool:
                 and not envs.is_set("VLLM_ATTENTION_BACKEND")
             ) or envs.VLLM_ATTENTION_BACKEND == "FLASH_ATTN_VLLM_V1"
             supported = False
-            if fp8_attention and will_use_fa:
+            if current_platform.is_rocm():
+                supported = True
```

Comment:
what does "supported" mean here? I thought that this check is whether the kernel supports FP8 attention (e.g., query dtype in fp8) rather than only KV cache in fp8.

Reply:
no that's wrong, please ignore. :)